snapshot-integration/readme.md

```diff
@@ -1,36 +1,38 @@
 # ThousandEyes Snapshot Integration
-Create a ThousandEyes snapshot when an AppDynamics Event / Alert occurs.
 
-This uses AppDyamics' HTTP Request Template webhook integration feature, similar to ThousandEyes alert webhook notification. 
+Create a ThousandEyes snapshot when an AppDynamics event or alert occurs.
 
-For each ThousandEyes test associated with a given AppD application (or tier / node) we will create a separate HTTP request to trigger a ThousandEyes snapshot via the ThousandEyes API. 
+This uses AppDynamics' HTTP Request Template webhook integration feature, similar to the ThousandEyes alert webhook notification. 
 
-We will create an HTTP Request template so that the ThousandEyes snapshot time window and TestID can be provided dynamically.
+For each ThousandEyes test associated with a given AppDynamics application (or tier or node), we will create a separate HTTP request to trigger a ThousandEyes snapshot via the ThousandEyes API. 
```

Comment:
```suggestion
Once created, our HTTP request template can be called by Actions defined in AppDynamics. We will define dynamic variables for the template - `testId`, `testName`, and `accountId` - that are specified by each Action, so that a single HTTP request template can be used to trigger a snapshot for any test in ThousandEyes. 
```